source/baseObject.py

```diff
@@ -133,7 +133,11 @@ class ScriptableType(AutoPropertyType):
 
 	def __new__(meta, name, bases, dict):
 		cls = super(ScriptableType, meta).__new__(meta, name, bases, dict)
-		gestures = getattr(cls, "_%s__gestures" % cls.__name__, {})
+		# #8463: To avoid name mangling conflicts, create a copy of the __gestures dictionary.
+		try:
+			gestures = getattr(cls, "_%s__gestures" % cls.__name__).copy()
+		except AttributeError:
+			gestures = {}

```

Comment:
I've tried to explain this in a comment now. This should happen when a user doesn't override the _-gestures dictionary, so actually it will be the case for most objects by default. I wonder whether a check using hasattr is more efficient than using an exception handler.